modules/standard/Path.chpl

```diff
@@ -663,48 +612,36 @@ proc realPath(path: string): string throws {
   return ret; 
 }
 
-pragma "no doc"
-pragma "last resort"
-proc realPath(name: string): string throws {
-  compilerWarning("Path.realPath: Argument 'name' is deprecated - use 'path' instead");
-  return realPath(path=name);
-}
-
-/* Determines the canonical path referenced by the :type:`~IO.file` record
+/* Determines the canonical path referenced by a given :type:`~IO.file` record
    performing this operation.  This resolves and removes any :data:`curDir` and
    :data:`parentDir` uses present, as well as any symbolic links.  Returns the
    result.
 
-   :return: A canonical path to the file referenced by this :type:`~IO.file`
+   :arg f: A file whose path to resolve.
```

Comment:
This sentences feels like there's a word or two missing, but the exact words would be something "you wish" or "we desire" and I don't think that fits the tone we try to use for documentation (though I may be misremembering).  I think maybe it should say "A file whose path should be resolved" instead?

Reply:
I have made the adjustment you suggested. I also see some "desired"-type wording, which I don't like as you've alluded to.

Personally, I think we can drop most of the argument and return documentations here without losing value. Because the description texts are plenty clear in terms of requirements of these arguments. But that's also part of the question of "what's the tone (or degree of formality) in our docs?", and I can see the opposite side of my point. Another question is about the consistency across docs. Because obviously we need to document returns and arguments in some functions. And then, where do we draw the line for "some"?

Reply:
Yeah.  I like having them and for me always adding documentation like that makes it easier to remember to add documentation for them when it is important, but it does seem like an undue burden to request of others.  It'd be good to figure out what the guidance should be, though